src/main/kotlin/com/autonomousapps/tasks/AbiAnalysisTask.kt

```diff
@@ -66,6 +74,9 @@ abstract class AbiAnalysisWorkAction : WorkAction<AbiAnalysisParameters> {
     // Inputs
     val jarFile = parameters.jar.get().asFile
     val dependencies = parameters.dependencies.get().asFile.readText().fromJsonList<Component>()
+    val exclusions = parameters.exclusions.takeIf { it.isPresent }?.run {
```

Comment:
I have a preference for
```kotlin
val exclusions = parameters.exclusions.orNull?.run {
  fromJson<AbiExclusions>()
} ?: AbiExclusions.NONE
```
to match the existing pattern. It's also a _tiny_ bit less verbose.

Reply:
TIL orNull!

Reply:
0e501e0